community/modules/scheduler/schedmd-slurm-gcp-v6-controller/slurm_files.tf

```diff
@@ -120,7 +120,7 @@ module "slurm_files" {
   prolog_scripts           = var.prolog_scripts
   enable_slurm_gcp_plugins = var.enable_slurm_gcp_plugins
 
-  disable_default_mounts = var.disable_default_mounts
+  disable_default_mounts = !var.enable_default_mounts
```

Comment:
Are we planning to update this as well  ? All the way to slurm-gcp ?

Reply:
This is a part of SlurmGCP modules , we will address them when/if we will merge them with wrappers.